sdk/eventgrid/Azure.Messaging.EventGrid/tests/ConsumeEventTests.cs

```diff
@@ -104,8 +104,14 @@ public void ConsumeEventUsingBinaryDataExtensionMethod()
         [Test]
         public void EGEventGetDataThrowsWhenCalledWithoutParse()
         {
+            var customSerializer = new JsonObjectSerializer(
+                new JsonSerializerOptions()
+                {
+                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+                });
+
             EventGridEvent egEvent = new EventGridEvent(new ContosoItemReceivedEventData(), "", "Contoso.Items.ItemReceived", "1");
-            Assert.That(() => egEvent.GetData<ContosoItemReceivedEventData>(),
+            Assert.That(() => egEvent.GetData<ContosoItemReceivedEventData>(customSerializer),
```

Comment:
Can we also add in a GetData calls without the serializer (both generic and non-generic)?